src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasInMemoryService.kt

```diff
@@ -64,7 +67,7 @@ class CosmasService : CosmasGrpc.CosmasImplBase() {
 
     override fun createVersion(request: CosmasProto.CreateVersionRequest,
                                responseObserver: StreamObserver<CosmasProto.CreateVersionResponse>) {
-        println("Get request for create new version")
+        println("Get request for create new version of file № ${request.fileId}")

```

Comment:
Using `№` is quite uncommon for indicating id or number. Programmers are more likely to see and understand `#`, so let's use it.

Reply:
done